modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -2,137 +2,312 @@
 ----
-podman login -u kubeadmin -p $(oc whoami -t) $REGISTRY_HOST
+ls <workspace-directory>/<results-directory>
 ----
-====
++
+--
+where:
 
-. Log in to the `registry.redhat.io` in `podman` by running the following command:
+`<workspace-directory>` :: Specifies the name of your workspace directory, for example, `oc-mirror-workspace`.
+
+`<results-directory>` :: Specifies the name of your results directory, for example, `results-1738070846`.
+--
 +
-[source,terminal]
+. To mirror the Helm chart, deploy the `imageContentSourcePolicy.yaml` file in the disconnected cluster by running the `oc apply` command. For example:
++
+[source,terminal,subs="attributes+"]
 ----
-podman login registry.redhat.io
+oc apply -f <workspace-directory>/<results-directory>/<image-content-policy-file>
 ----
 +
-For more information about registry authentication, see https://access.redhat.com/RegistryAuthentication[{company-name} Container Registry Authentication].
+--
+where:
+
+`<workspace-directory>` :: Specifies the name of your workspace directory, for example, `oc-mirror-workspace`.
 
-. Pull {product-short} and PostgreSQL images from https://catalog.redhat.com/software/containers/search[{company-name} Image registry] to your workstation, by running the following commands:
+`<results-directory>` :: Specifies the name of your results directory, for example, `results-1738070846`.
+
+`<results-directory>` :: Specifies the name of the image content source policy file, for example, `ImageContentSourcePolicy.yaml `.
+--
+. In your air-gapped environment, deploy the Helm chart to the namespace that you want to use by running the `helm install` command and `namespace` options. For example:
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal,subs="attributes+"]
 ----
-podman pull registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-version}
+helm install <rhdh-instance> <workspace-directory>/<results-directory>/<charts-directory>/<archive-file> --namespace <your-namespace> --create-namespace
 ----
 +
-[source,terminal,source,subs="attributes+"]
+--
+where:
+
+`<rhdh-instance>` :: Specifies the name of your {product} instance, for example, `my-rhdh`
+
+`<workspace-directory>` :: Specifies the name of your workspace directory, for example, `oc-mirror-workspace`.
+
+`<charts-directory>` :: Specifies the name of the charts directory, for example, `charts`.
+
+`<results-directory>` :: Specifies the name of your results directory, for example, `results-1738070846`.
+
+`<archive-file>` :: Specifies the name of the archive file containing the resources that you want to mirror, for example, `redhat-developer-hub-1.4.1.tgz`.
+
+`<your-namespace>` :: Specifies the namespace that you want to deploy the Helm chart to, for example, `{my-product-namespace}`.
+--
+
+.Verification
+. To verify that all resources are successfully deployed and running, check the status of your deployments and pods in the cluster.
+. View the router base URL by entering the following command:
```

Comment:
IMO, this step is not needed if the `clusterRouterBase` is passed in the previous `helm install` step.